index.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+const greet = require('./lib/greet');
+// const argv = require('argv');
```

Comment:
I find comments helpful, but try getting into the habit of taking out commented code prior to pushing your final assignment submission.